core/inherents/src/lib.rs

```diff
@@ -153,7 +153,7 @@ impl codec::Decode for InherentData {
 ///
 /// When a fatal error occurres, all other errors are removed and the implementation needs to
 /// abbort checking inherents.
-#[derive(Encode, Decode, Clone)]
```

Comment:
I'm actually pretty hesitatnt on this one - Seems that `CheckInherentsResults::new`and `CheckInherentsResults::default` return different values which might be quite misleading.

Reply:
Fuu, good catch..